deploy/helm/stac-scaler/templates/deployment.yaml

```
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment-app-stacfastapi
+  namespace: {{ $.Values.namespace }}
```

```suggestion
  namespace: {{ .Values.namespace }}
```
The `$` is unnecessary here, but see my comments above. I think this should be changed. A typical chart would have `namespace: {{ .Release.Namespace }}` here to deploy in the namespace of the helm chart itself.

---

Addressed.